redshift_connector/cursor.py

```diff
@@ -444,10 +474,12 @@ def write_dataframe(self: "Cursor", df: "pandas.DataFrame", table: str) -> None:
         try:
             import pandas
         except ModuleNotFoundError:
-            raise ModuleNotFoundError(MISSING_MODULE_ERROR_MSG.format(module="pandas"))
+            raise ModuleNotFoundError(
+                MISSING_MODULE_ERROR_MSG.format(module="pandas"))
 
         if not self.__is_valid_table(table):
-            raise InterfaceError("Invalid table name passed to write_dataframe: {}".format(table))
+            raise InterfaceError(
+                "Invalid table name passed to write_dataframe: {}".format(table))
```

Comment:
@Yash621 is this necessary? adds noise to the PR.

Reply:
fixed in revisions